moto/__init__.py

```diff
@@ -167,6 +167,8 @@ def f(*args, **kwargs):
 mock_xray_client = lazy_load(".xray", "mock_xray_client")
 mock_wafv2 = lazy_load(".wafv2", "mock_wafv2")
 mock_textract = lazy_load(".textract", "mock_textract")
+mock_identitystore = lazy_load(".identitystore", "mock_identitystore", boto3_name="identitystore")
```

Comment:
The boto3-name will be automatically deduced in most cases, so we don't have to explicitly define it
```suggestion
mock_identitystore = lazy_load(".identitystore", "mock_identitystore")
```
We also only have to define it once :slightly_smiling_face: